### PR TITLE
Pinned: message changed

### PR DESCRIPTION
**Pull request by puneeth-chanda**
Pull Request for Issue # 21879

### Summary of Changes:
The pinned message which is too long to read is replaced by a condensed text and a help page URL.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_banners.ini [hunk 1]
```diff
@@ -95,7 +95,7 @@ COM_BANNERS_FIELD_NAME_LABEL="Name"
 COM_BANNERS_FIELD_PUBLISH_DOWN_LABEL="Finish Publishing"
 COM_BANNERS_FIELD_PUBLISH_UP_LABEL="Start Publishing"
 COM_BANNERS_FIELD_PURCHASETYPE_LABEL="Purchase Type"
-COM_BANNERS_FIELD_STICKY_DESC="Whether or not the Banner is 'pinned'. If one or more Banners in a Category are pinned, they will take priority over Banners that are not pinned. For example, if two Banners in a Category are pinned and a third Banner is not pinned, the third Banner will not display if the module setting is 'Pinned, Randomise'. Only the two pinned Banners will display."
+COM_BANNERS_FIELD_STICKY_DESC="Take priority over not pinned Banners.see this:  https://docs.joomla.org/Banner."
```
Comment (infograf768) [inline administrator/language/en-GB/en-GB.com_banners.ini:98]: English would have to be modified, but worse imho is the fact that nowhere in https://docs.joomla.org/Banner or even help is explained precisely what "pinned" does as it depends  on the module settings...

**Comment by infograf768** [comment]
@puneeth2001 
Your last changes do not concern the original issue and my remarks were not considered.

Test negative for me.

**Comment by puneeth-chanda** [comment]
sorry, but I am trying to find the help page URL but couldn't find it so I moved on to next issue. If you have the link please share it with me so that I can modify the necessary changes.

**Comment by wojsmol** [comment]
@puneeth2001 Please revert latest commit and then edit installation/language/en-GB/en-GB.ini as required.

**Comment by infograf768** [comment]
a url in a tip is anyway useless i am afraid as it can’t be clicked or copied.
I think this pr is useless.

**Comment by puneeth-chanda** [comment]
Then can I only put a help text instead of link

**Comment by infograf768** [comment]
we already have infos with the existing string value. Why change?

**Pull request closed without merging**